operations/patch_util.go

```diff
@@ -622,21 +622,16 @@ Continue?`, uncommittedChangesFlag), true), nil
 // The branch argument is used to determine where to generate the merge base from, and any extra
 // arguments supplied are passed directly in as additional args to git diff.
 func loadGitData(dir, remote, branch, ref, commits string, format bool, extraArgs ...string) (*localDiff, error) {
-	if remote == "" {
-		remote = "upstream"
-	}
-	// remote/branch refers directly to the remote branch and does not require a local branch.
-	// branch@{remote} refers to the remote branch that "branch" is tracking.
+	// branch@{upstream} refers to the branch that the branch specified by branchname is set to
+	// build on top of. This allows automatically detecting a branch based on the correct remote,
```

Comment:
nit:  This seems confusing. Could we say something like `for example, if a user's repo is a fork, this allows automatic detection....`